src/CommandAndStateMessageParser.cpp

```diff
@@ -370,6 +371,15 @@ double CommandAndStateMessageParser::getCpuTemperature(string address)
     return mJData["payload"]["devices"][address]["cpuTemp"].asDouble();
 }
 
+base::samples::RigidBodyState CommandAndStateMessageParser::getCameraHeadTiltPosition(string address)
+{
+    m_camera_head_tilt_position.orientation = Quaterniond(
+        AngleAxisd(mJData["payload"]["devices"][address]["cameraHead"]["tilt"]["position"]
+                       .asDouble(),
+            Vector3d::UnitZ()));
+    return m_camera_head_tilt_position;
```

Comment:
In here, you will use `m_camera_tilt_position` to compute the orientation. Keep in mind the parameter limit, which are apparently (please confirm) between -110 and 90.